aziot/src/init.rs

```diff
@@ -1,15 +1,18 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-// This subcommand interactively asks the user to give out basic provisioning information for their device and
-// creates the config files for the three services based on that information.
-//
-// Notes:
-//
-// - Provisioning with a symmetric key (manual or DPS) requires the key to be preloaded into KS, which means it needs to be
-//   saved to a file. This subcommand uses a file named `/var/secrets/aziot/keyd/device-id` for that purpose.
-//   It creates the directory structure and ACLs the directory and the file appropriately to the KS user.
-//
-// - `dynamic_reprovisioning` is enabled by default in IS provisioning settings.
+/// This subcommand interactively asks the user to give out basic provisioning information for their device and
```

Comment:
uhh, this is now a doc-comment on `const DPS_GLOBAL_ENDPOINT`.

I think you meant to use `//!`

Reply:
Derp.

Reply:
nice commit message lol